commands/describe_test.go

```diff
@@ -0,0 +1,29 @@
+package commands
+
+import "testing"
+
+func Test_getFunctionURLs(t *testing.T) {
+	cases := []struct {
+		name              string
+		gateway           string
+		functionName      string
+		functionNamespace string
+		expectedURL       string
+		expectedAsyncURL  string
+	}{
+		{"localhost", "http://127.0.0.1:8080", "figlet", "alpha", "http://127.0.0.1:8080/function/figlet.alpha", "http://127.0.0.1:8080/async-function/figlet.alpha"},
+		{"secure site", "https://example.com", "nodeinfo", "beta", "https://example.com/function/nodeinfo.beta", "https://example.com/async-function/nodeinfo.beta"},
+		{"no namespace", "https://example.com:31112", "nodeinfo", "", "https://example.com:31112/function/nodeinfo", "https://example.com:31112/async-function/nodeinfo"},
+	}
+
+	for _, tc := range cases {
+		t.Run(tc.name, func(t *testing.T) {
+			url, asyncURL := getFunctionURLs(tc.gateway, tc.functionName, tc.functionNamespace)
+
+			if url != tc.expectedURL || asyncURL != tc.expectedAsyncURL {
+				t.Fatalf("incorrect URL(s):\n got %s and %s\n expected %s and %s\n", url, asyncURL, tc.expectedURL, tc.expectedAsyncURL)
```

Comment:
Please use verbiage of want/got instead

I think we can do without the newlines too, perhaps use `%q` if you need to see the exact quoted values.

Reply:
Cool, thanks for the tip, I'll fix this Monday.

Reply:
Done.